### PR TITLE
Header display issues on mobile

### PR DESCRIPTION
**Pull request by sebastian-ew**
## Description

Adds some styles to fix issues with home menus and internal pages breadcrumbs.

## Related ticket

(If a ticket in your task manager is linked to this PR, replace this paragraph
with the URL to the ticket. If not applicable, you can remove this whole
section.)

> Make sure to read the *Pull requests process* in CONTRIBUTING.md.
> 
> Main points:
> 
> * The assigned reviewer(s) must always *submit* a review (not simply add a
>   comment).
>
> * If the PR is "Approved", the creator of the PR must then:
>   * Implement the changes in the reviewer's comments, if applicable, and mark
>     them as "Resolved".
>   * Squash and merge the branch.
>   * Delete the branch.
> 
> * If the PR is "Request Changes", the creator of the PR must then:
>   * Update the code as requested.
>   * Re-request a review from the reviewer.
>   * NOT mark the comments as "Resolved", the reviewer will.
> 
> (You can keep or remove this block, as you wish.)

### CODE REVIEW

File: fw-child/resources/scss/child.scss [hunk 1]
```diff
@@ -441,6 +441,19 @@ $control-slider-handle-width: 2.75rem;
 
 // BREADCRUMB
 
+#header-breadcrumb {
+  overflow-x: scroll;
```
Comment (xfrenette-crim) [inline fw-child/resources/scss/child.scss:445]: On Chrome Windows, this CSS instruction has the effect of always showing a scrollbar.

image

File: fw-child/resources/scss/_variables.scss [hunk 2]
```diff
@@ -1604,10 +1604,10 @@ $figure-caption-color:              var(--#{$prefix}secondary-color) !default;
 // Breadcrumbs
 
 // scss-docs-start breadcrumb-variables
-$breadcrumb-font-size:              1.25rem;
+$breadcrumb-font-size:              1rem;
```
Comment (xfrenette-crim) [inline fw-child/resources/scss/_variables.scss:1607]: The breadcrumb size should be reduced only on smaller screens. On bigger screens, the original size of `1.25rem` was more appropriate.

Suggestion: add custom rules for `.breadcrumb` in `_vendor.scss`.

**Review by xfrenette-crim — CHANGES_REQUESTED** [review]

File: fw-child/resources/scss/child.scss (hunk 1 above)
Comment (sebastian-ew) [reply fw-child/resources/scss/child.scss]: @xfrenette-crim this was changed to not scroll on desktop. Thanks!

File: fw-child/resources/scss/_variables.scss (hunk 2 above)
Comment (sebastian-ew) [reply fw-child/resources/scss/_variables.scss:1607]: @xfrenette-crim rule for desktop breadcrumbs was added. Thanks!

**Review by xfrenette-crim — APPROVED** [review]

**Pull request merged**